src/items/class-features/burn_enchantment_(su).json

```diff
@@ -0,0 +1,148 @@
+{
+  "_id": "FfoYHZQ6GS6YjiT9",
+  "name": "Burn Enchantment (Su)",
+  "type": "feat",
+  "img": "systems/sfrpg/images/cup/gameplay/photon.webp",
+  "system": {
+    "type": "",
+    "ability": "",
+    "abilityMods": {
+      "parts": []
+    },
+    "actionTarget": "",
+    "actionType": "other",
+    "activation": {
+      "type": "move",
+      "condition": "",
+      "cost": null
+    },
+    "area": {
+      "effect": "",
+      "shape": "",
+      "units": "none",
+      "value": null
+    },
+    "attackBonus": 0,
+    "chatFlavor": "",
+    "critical": {
+      "effect": "",
+      "parts": []
+    },
+    "damage": {
+      "parts": [
+        {
+          "name": "",
+          "formula": "(lookupRange(@classes.solarian.levels, 0, 2, 1, 6, 2, 10, 3, 14, 4, 18, 5))d6",
+          "group": null,
+          "types": {
+            "acid": false,
+            "bludgeoning": false,
+            "cold": false,
+            "electricity": false,
+            "fire": true,
+            "healing": false,
+            "piercing": false,
+            "slashing": false,
+            "sonic": false
+          }
+        }
+      ]
+    },
+    "damageNotes": "",
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.G7p2YIGFTAU6uGjL]{Stellar Revelation}</p>\n<p><sub>&nbsp;</sub>@Icon[type:photon]Photon Revelation</p>\n<p>Your attacks’ cosmic energy shreds the magical forces wielded by spellcasters. As a move action, you sheathe your body in protective energy that attunes to spellcasters you strike. If the next successful melee attack you make before the beginning of your next turn hits a creature capable of casting spells or using spell-like abilities, you gain a +2 enhancement bonus to saving throws against that creature’s spells and spell-like abilities for 1 round.</p>\n<p>When you are attuned or fully attuned, your weapon gains the dispelling critical effect. If the weapon already has a critical hit effect, when you score a critical hit, you can apply either the weapon’s normal critical hit effect or the dispelling effect. A target affected by the dispelling critical hit effect is the subject of a targeted @UUID[Compendium.sfrpg.spells.KQ4vSfG9tq8EJxir]{Dispel Magic}, per the spell, using your solarian level as the caster level.</p>"
```

Comment:
Spells should be italicized. Italicize the link to dispel magic. Same applies to other spells